typescript/packages/plugins/modespray/src/parameters.ts

```diff
@@ -0,0 +1,16 @@
+import { z } from "zod";
+
+export const DisperseEtherParameters = z.object({
+    recipients: z
+        .array(z.string().regex(/^0x[a-fA-F0-9]{40}$/, "Invalid address"))
+        .nonempty(),
+    amounts: z.array(z.string().regex(/^\d+$/, "Invalid amount")).nonempty(),
```

Comment:
I would specify also in the description that amounts are in basis units

Reply:
updated!